modules/virt-opting-out-workload-updates.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-opting-out-workload-updates_{context}"]
+= Opting out of automatic workload updates before you upgrade to {VirtProductName} {VirtVersion}
+
+Automatic workload updates are enabled by default as of {VirtProductName} {VirtVersion}. If you plan to disable this feature, you can do so _before_ upgrading by following this workaround.
+
+[NOTE]
+====
+If you already upgraded to {VirtProductName} {VirtVersion}, you can disable automatic workload updates by setting the value of `spec.workloadUpdateStrategy.workloadUpdateMethods` to `[]` in the `HyperConverged` CR.
+====
+
+.Prerequisites
+
+* You have not yet upgraded to {VirtProductName} {VirtVersion}.
+
+.Procedure
+
+. Annotate the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-cnv annotate hco kubevirt-hyperconverged --overwrite kubevirt.kubevirt.io/jsonpatch='[{"op": "replace", "path": "/spec/workloadUpdateStrategy/workloadUpdateMethods", "value": []}]'
+----
+
+. Upgrade {VirtProductName} to version {VirtVersion}. If you use the *Automatic* approval strategy, upgrading {product-title} triggers the {VirtProductName} update. Ensure that the {VirtProductName} upgrade completes before moving to the next step.
+
+. Patch the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-cnv patch hco kubevirt-hyperconverged --type='json' -p='[{"op": "replace", "path": "/spec/workloadUpdateStrategy/workloadUpdateMethods", "value":[]}]'
```

Comment:
Would it make sense to break up the three long commands in this procedure into multiple lines by using a backslash?

Reply:
Because we have scrollbars and a copy button, I think it's okay to keep them as they are.